commit d921f30bbb75faca11e907cf421a2a2f68dc300b
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Thu Aug 23 15:13:55 2018 +0100

    ARM: dts: bcm283x: Correct mailbox register sizes
    
    The size field in a Device Tree "reg" property is encoded in bytes, not
    words.
    
    Signed-off-by: Phil Elwell <phil@raspberrypi.org>

diff --git a/arch/arm/boot/dts/bcm2835-rpi.dtsi b/arch/arm/boot/dts/bcm2835-rpi.dtsi
index cb2d6d78a7fb..c481eab1bd7c 100644
--- a/arch/arm/boot/dts/bcm2835-rpi.dtsi
+++ b/arch/arm/boot/dts/bcm2835-rpi.dtsi
@@ -32,7 +32,7 @@
 
 		mailbox@7e00b840 {
 			compatible = "brcm,bcm2835-vchiq";
-			reg = <0x7e00b840 0xf>;
+			reg = <0x7e00b840 0x3c>;
 			interrupts = <0 2>;
 		};
 	};
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1519_d921f30bbb75faca11e907cf421a2a2f68dc300b.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1519_d921f30bbb75faca11e907cf421a2a2f68dc300b.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
